modules/custom/az_paragraphs/az_paragraphs.install

```diff
@@ -15,6 +15,7 @@ function az_paragraphs_update_920301(&$sandbox) {
 
   if (!isset($sandbox['total'])) {
     $count = \Drupal::entityQuery('paragraph')
+      ->accessCheck(FALSE)
```

Comment:
Was this on purpose?

Reply:
NVM I See you put 

> Added explicit access checks to entity queries for [drupal.org/node/3201242](https://www.drupal.org/node/3201242) (deprecation introduced in Drupal 9.2.x)

Reply:
NVM I see this now 

> Addressed Twig deprecations (see [twig.symfony.com/doc/2.x/deprecated.html#tags](https://twig.symfony.com/doc/2.x/deprecated.html#tags))
Adding an if condition on a for tag is deprecated in Twig 2.10. Use a filter filter or an "if" condition inside the "for" body instead (if your condition depends on a variable updated inside the loop).

Reply:
Yes.  See the updated issue description.